client/components/external-link/README.md

```diff
@@ -12,7 +12,15 @@ import ExternalLink from 'components/external-link';
 
 class MyComponent extends React.Component {
 	render() {
-		return <ExternalLink icon={ true } href="https://wordpress.org" onClick="somefunction()">WordPress.org</ExternalLink>;
+		return (
+			<ExternalLink
+				icon={ true }
+				href="https://wordpress.org"
+				onClick={ () => somefunction() }
```

Comment:
`somefunction()` still does nothing. Let's make it an empty function so the example is valid:

```suggestion
				onClick={ () => {} }
```